model/genere.rb

```diff
@@ -0,0 +1,14 @@
+class Genere
+  attr_accessor :name
+
+  def initialize(name, id = Random.rand(1..999_999))
+    @id = id
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
+    @items.push(item)
+    item.genere = self
```

Comment:
well done in this class, however, in the diagram class, it's named "genre" instead "genere", please change the name, in this class and in the name of the file, also in item file.

Reply:
Thanks for mentioning